app/javascript/apiService.js

```diff
@@ -102,11 +102,9 @@ const deleteRentalAssistance = async (rentalAssistanceId) => {
   return request.destroy(`/rental-assistances/${rentalAssistanceId}`)
 }
 
-export const createOrUpdateLease = async (lease, applicationId) => {
-  const leaseId = lease['id']
-  const data = {
-    lease: lease
-  }
+export const createOrUpdateLease = async (leaseToCreateOrUpdate, applicationId) => {
```

Comment:
this looks like changes that bled over from your previous PR?

Reply:
Not sure what you mean here, I updated this so that the variable `lease` was not being used so many times for potential side effects

Reply:
ok thanks for the clarification! i wasn't sure if it was a rebase issue or something new.